internal/util/util.go

```diff
@@ -445,14 +503,9 @@ func Clamp(val, min, max int) int {
 	return val
 }
 
-// IsNonAlphaNumeric returns if the rune is not a number of letter or underscore.
-func IsNonAlphaNumeric(c rune) bool {
-	return !unicode.IsLetter(c) && !unicode.IsNumber(c) && c != '_'
-}
-
 // IsAutocomplete returns whether a character should begin an autocompletion.
 func IsAutocomplete(c rune) bool {
-	return c == '.' || !IsNonAlphaNumeric(c)
+	return c == '.' || !IsNonWordChar(c)
```

Comment:
As above `IsWordChar(c)` instead of `!IsNonWordChar(c)`?

Reply:
+1

Reply:
Yeah, that's better.

> Definitely 👍 for proposing this feature and thanks to you guys for taking care of creation and review.

You are welcome. And let me take advantage of the situation and say: Thank you guys for your general commitment to this project. 👍

> ... , but was already in the situation to miss something like that. :)

And I started to believe I am to the only one using sub-word navigation :smile: